pkg/commands/game/mute_vc.go

```diff
@@ -0,0 +1,150 @@
+package game
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/itfactory-tm/thomas-bot/pkg/embed"
+
+	"github.com/itfactory-tm/thomas-bot/pkg/command"
+
+	"github.com/bwmarrin/discordgo"
+	"github.com/itfactory-tm/thomas-bot/pkg/sudo"
+)
+
+// adduser contains the bob!adduser and bob!remuser command
+type MuteCommand struct{}
+
+// NewMuteCommand gives a new MuteCommand
+func NewMuteCommand() *MuteCommand {
+	return &MuteCommand{}
+}
+
+// Register registers the handlers
+func (m *MuteCommand) Register(registry command.Registry, server command.Server) {
+	registry.RegisterMessageCreateHandler("mutevc", m.mutevc)
+	registry.RegisterMessageReactionAddHandler(m.handleMuteReaction)
+}
+
+func (m *MuteCommand) mutevc(s *discordgo.Session, msg *discordgo.MessageCreate) {
+	if !sudo.IsItfAdmin(msg.Author.ID) {
+		s.ChannelMessageSend(msg.ChannelID, fmt.Sprintf("%s is not in the sudoers file. This incident will be reported.", msg.Author.ID))
+		return
+	}
+
+	// Get the guild status
+	g, err := s.State.Guild(msg.GuildID)
+	if err != nil {
+		s.ChannelMessageSend(msg.ChannelID, fmt.Sprintf("Error getting guild: %v", err))
+		return
+	}
+
+	channelId := strings.TrimPrefix(msg.Content, "bob!mutevc")
+	if len(channelId) > 1 {
+		channelId = strings.TrimPrefix(channelId, " ")
+		//ChannelID specified, check if channel exists
+		for _, channel := range g.Channels {
+			if channel.ID == channelId && channel.Type == 2 {
+				embedMsg, _ := s.ChannelMessageSendEmbed(msg.ChannelID, m.muteMenu(channel.ID, s).MessageEmbed)
```

Comment:
we should handle this error, if this fails to send the next use of the ID might cause a crash